src/pip/_internal/download.py

```diff
@@ -532,31 +538,54 @@ def suppressed_cache_errors():
     """
     try:
         yield
-    except (LockError, OSError, IOError):
+    except (OSError, IOError):
         pass
 
 
-class SafeFileCache(FileCache):
+class SafeFileCache(BaseCache):
     """
     A file based cache which is safe to use even when the target directory may
     not be accessible or writable.
```

Comment:
Looks like the phrase "which is safe to use even when the target directory may not be accessible or writable" is not actual since #6845.

Reply:
It should be. Any `IOError` or `OSError` will get swallowed up by `with suppressed_cache_errors(): ...`.

Reply:
Sorry, overlooked this context manager.